apps/web-staking/src/app/components/dashboard/DashboardStakingInfo.tsx

```diff
@@ -115,6 +113,9 @@ const DashboardStakingInfo = ({
     }
   }, [isSuccess, isError, updateOnSuccess, updateOnError]);
 
+  useEffect(() => {
+      loadRedemptions();
+  }, [claimable, loadRedemptions]);
```

Comment:
What is the purpose of this useEffect ?

Building the flow with useEffect triggers does not only slow down your app, it will lead to uncontrolled calls. This is especially dangerous if you are doing blockchain reads you can get RPC blocked real quick and we should not be providing a private RPC for the staking app.

Reply:
Removing the useEffect here prevents the redemptions from being loaded. 

The getRedemptionsHook includes a variable to track the timestamp of the last blockchain call. We can set that value to anything we choose(currently 1 second) to prevent excess calls. 

I've tested the current implementation and confirmed it only makes a single RPC call per 500 redemptions. Monitoring the console log and network tab confirms. 

If additional changes are requested, just let me know!

Reply:
> Update: the issue of `Removing the useEffect here prevents the redemptions from being loaded.` was because I didn't have the sepolia configs. Once I updated the Sepolia configs, the redemptions loaded without the useEffect. It has been removed.

Im concerned by this comment, what Sepolia config needed to be added for web-staking ?